### ISSUE TITLE
Better support for multihead sim racing 

### ISSUE THREAD

**Issue by addeps3** [ValveSoftware/Proton#5213]
# Feature Request

First of all, the Linux gaming community _really_ appreciates all the work that has been done with proton.

Since  long ago, the typical way for wine-users with 2 or more monitors to play with 3840/5760 screenwidth has been to enable the virtual desktop with desired screenwidth and change the game resolution with the in-game options. 

Since proton 5.13(?) and the fullscreen hack and AMD fsr patch where added(unless Im mistaken with the latter) this way for extra screenwidth has not been possible. 
It is possible to use proton 5.0, but this version is according to some users "slower" than the more recent ones, and sim racers and I often want to use the fastest proton.

Somebody requested a option to disable the FS hack in runtime, but my request is different in that multihead users, and the sim racing community still wish for the opportunity to be able to game on later proton versions. 

## I confirm:
- [ x] that I haven't found another request for this feature.
- [x ] that I have checked whether there are updates for my system available that
      contain this feature already.

## Description  What do you want to be added? 

At the very least; offer config-instructions on github for building a customized, and functional proton with these hacks disabled. 
This would kind of create two protons, but its cost-effective.

At the most; make proton work for both multihead users and single screen. Dont know if this would disable fsr_hack for all games though. 

## Justification [optional]  Why integration in Proton instead of Wine? 

This might be an issue for wine.

## Risks [optional]
asteroid impact (joke, sorry)

**Comment by kisak-valve** [comment]
Hello @addeps3,

The issue with using wine's virtual desktop config with Proton 5.13 is being tracked at https://github.com/ValveSoftware/Proton/issues/4673.
An option to disable Proton's fullscreen hack isn't going to happen (https://github.com/ValveSoftware/Proton/issues/2831#issuecomment-505005893)
FSR integration is currently under evaluation and is unrelated (https://github.com/ValveSoftware/wine/pull/116)
Building specialized variants of Proton is not sustainable and not going to happen for practical reasons.

Setting those details aside, let's use this feature request as the general request to have the ability for a game to span multiple monitors, with no fancy monitor layout reconfiguration beyond maybe having something like `PROTON_ALL_DISPLAYS=1` as a launch option. It should be noted that if something like Xwayland is also thrown into the mix, that would also need to span all displays before Proton is run inside it, or for upstream wine to integrate and expose the needed functionality to do this with wayland directly.

**Comment by addeps3** [comment]
@kisak-valve 
I agree, and appreciate your open-mindedness about this feature-request.

**Comment by bobbadshy** [comment]
On Linux Mint, when using the (agreeably quite outdated, but still working ;) `compiz` display manager) it is possible to force the fullscreen desktop area to span all monitors of a multi-monitor setup. So, `compiz --replace`, in combination with above suggested Proton switch (to report full, all monitor desktop resolution to wine as display resolution) would enable true fullscreen Proton multi monitor support under X11.

**Comment by zzhiyi** [comment]
What game are you talking about specifically? The fullscreen hack of proton 6.3-8 should support multi-monitor now. If it's not, then there is a bug.

**Comment by laxdragon** [comment]
> What game are you talking about specifically? The fullscreen hack of proton 6.3-8 should support multi-monitor now. If it's not, then there is a bug.

Try either Project Cars 2 or Project Cars 3. You will need to enable triple screen in the settings. Once enabled, it squishes all 3 "displays" onto a single monitor instead of spanning across all 3.

**Comment by zzhiyi** [comment]
ok, I will take a look at this. I will need to get some hardware first.
